src/inc/internal/BundleWriterHelper.hpp

```diff
@@ -0,0 +1,95 @@
+#pragma once
+
+#include "AppxPackaging.hpp"
+#include "ComHelper.hpp"
+#include "DirectoryObject.hpp"
+#include "AppxBlockMapWriter.hpp"
+#include "ContentTypeWriter.hpp"
+#include "ZipObjectWriter.hpp"
+#include "BundleManifestWriter.hpp"
+#include "AppxPackageInfo.hpp"
+
+#include <map>
+#include <memory>
+#include <future>
+
+namespace MSIX {
+
+    struct PackageInfo
+        {
+            APPX_BUNDLE_PAYLOAD_PACKAGE_TYPE type;
+            UINT64 version;
+            std::string architecture;
+            std::string resourceId;
+            PCWSTR fileName;
+            UINT64 size;
+            UINT64 offset;
+            //ComPtr<IAppxManifestQualifiedResourcesEnumerator> resources;
+            ComPtr<IAppxManifestResourcesEnumerator> resources;
+
+            BOOL isDefaultApplicablePackage;

```

Comment:
>BOOL [](start = 12, length = 4)

use bool

Reply:
Removed this field as this is not being used anywhere as of now.